src/components/myfeelingspage/AddFeelings.tsx

```diff
@@ -4,8 +4,13 @@ import { useTranslation } from 'react-i18next';
 import { useRecoilValue } from 'recoil';
 
 import { HeaderDashboard } from '@components/dashboard/HeaderDashboard';
+<<<<<<< HEAD:src/components/myfeelingspage/AddFeelings.tsx
```

Comment:
Why are these conflict resolution marks here?

Reply:
I m surprised. I resolved them at my local machine and then pushed. The working tree is clear, no conflicts are At my end